### ISSUE TITLE
Add new signatures for Sync2Ad

### ISSUE THREAD

**Issue by kytta** [Exodus-Privacy/etip#166]
I was inspecting the TéléStar app, and I have discovered that Sync2Ad (https://etip.exodus-privacy.eu.org/trackers/b09cf757-bf4d-40df-aa81-ee31a9217a59/) has a new URL and thus can't be detected by exodus any more.

The new code signature would be: `tv.sync`, the correspoonding network signature would be `sync\.tv`

The old URL (sync2ad[dot]com) also redirtects to the new one, so I guess one can also update the tracker website

**Comment by kytta** [comment]
To see the differences between old and new packages, one can compare the TéléStar app versions [`2.10.8`](https://reports.exodus-privacy.eu.org/en/reports/66323/) (contains Sync2Ad) and [`2.12.20`](https://reports.exodus-privacy.eu.org/en/reports/154907/) (claims Sync2Ad is not there, but the package is now called `tv.sync.*`)

**Comment by pnu-s** [comment]
Thanks @kytta, I'm taking a look a this!

**Comment by pnu-s** [comment]
The [`2.12.20` version](https://reports.exodus-privacy.eu.org/en/reports/154907/) is pretty old (more than 2 years old), I took a look at the [`2.15.4` one](https://reports.exodus-privacy.eu.org/en/reports/345509/) and the packages `tv.sync.*` are not present.

Same goes for other apps which used to have the Sync2Ad tracker (ex: Europe1 app).

Are we sure about the new signature?

Or did these apps just remove the tracker?

**Comment by kytta** [comment]
> The [`2.12.20` version](https://reports.exodus-privacy.eu.org/en/reports/154907/) is pretty old (more than 2 years old), I took a look at the [`2.15.4` one](https://reports.exodus-privacy.eu.org/en/reports/345509/) and the packages `tv.sync.*` are not present.
> 
> Same goes for other apps which used to have the Sync2Ad tracker (ex: Europe1 app).
> 
> Are we sure about the new signature?
> 
> Or did these apps just remove the tracker?

Yes, seems like it. I am researching audio-based trackers at the moment, and they seem to not be as present as before. Sync on the whole hasn't had a blog update since 2019 afaik, so it can be that the company itself is not active any more. 

Yet, I am pretty sure about the name change. I decompiled the app with jadx and dug around for a bit, and the library still collects microphone data and all that, albeit with a different package architecture

**Comment by pnu-s** [comment]
Cool, thanks for your answer!

Would you have a more "precise" tracker code signature?
Such as `tv.sync.xxx`, to make sure we avoid raising false positives if some other packages signature are close.

**Comment by kytta** [comment]
> Would you have a more "precise" tracker code signature? Such as `tv.sync.xxx`, to make sure we avoid raising false positives if some other packages signature are close.

Yeah I figured this, too. Judging by the structure, we can be pretty sure that the apps including SYNC will have the packages `syncdisplay`, `syncmicrophone`, etc.:

<img width="197" alt="grafik" src="https://user-images.githubusercontent.com/6566248/235531608-488e5cca-44ae-4611-aaf1-476c0359b685.png">

Perhaps, `tv.sync.sync`? If it's regex, it will work.

**Comment by kytta** [comment]
@pnu-s sorry to ping you, but do you think you could merge this into Exodus and rescan the applications by ca. 26th May? I'd love to include more data in my thesis 😅
